<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueueWindow</class>
 <widget class="QWidget" name="QueueWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="dequeueButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>dequeue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enqueueButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>enqueue</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="item0">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="item1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="item2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="item3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="item4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="item5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="item6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="item7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="item8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="item9">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="modeComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="headLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>head</string>
   </property>
  </widget>
  <widget class="QLabel" name="tailLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>tail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemToBeEnqueue">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="queueSize">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
